doc/concepts/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _concepts:
+
+Concepts
+========
+
+Pachyderm enables you to create data pipelines that enable you
+continuously track changes that data scientists, developers, or other
+members of your team make to datasets. In other words, Pachyderm
+is a version-control platform for your data similar to what Git is
+for source code.
+
+Data scientints and researches often make changes to
+data in batches and phases. Often, they need to go back in time to
+revisit previous results or combine these data results with other
+data in different variants. Pachyderm helps solving the issues
+with reproducibility and data provenance by creating data pipelines
+which store the history of your changes from the begining of times.
+
+The two main Pachyderm concepts that
```

Comment:
PFS and PPS aren't really concepts. And "concepts encapsulating building blocks" doesn't really make sense. 

The pachyderm software has two main components, PFS and PPS. To be honest though, I actually don't think that users who aren't contributors looking at the code care or even make the distinction about PFS versus PPS. That's really a pachyderm internal abstraction that we should just remove entirely from docs. In our planning notes, we wrote them out that way, but really it should just be "versioned data concepts" (i.e. PFS) and "pipeline concepts" (i.e. PPS) or something like that.